### PR TITLE
Make Travis config use travis-perl helpers

### LINKED ISSUE #55: Include perl 5.26 in generated .travis.yml

### PR DESCRIPTION
**Pull request by mohawk2**
These changes turn inline-pm's CI from failing to passing (https://travis-ci.org/ingydotnet/inline-pm/builds/631915999) and also fix #55.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,18 +4,33 @@
...
-- '5.18'
-- '5.16'
-- '5.14'
-- '5.12'
-- '5.10'
-# - '5.8'
+matrix:
+  include:
+    # first as slowest
+    - perl: "5.26"
```
Comment (ingydotnet) [inline .travis.yml]: Stick with single quotes

File: .travis.yml [hunk 2]
```diff
@@ -35,15 +50,16 @@ install:
 #   Dist::Zilla::Plugin::TestMLIncluder
 #   Dist::Zilla::Plugin::TestML1Includer
 #   Dist::Zilla::PluginBundle::Git
+- [ ! -e test ] || cpan-install --coverage
```
Comment (ingydotnet) [inline .travis.yml]: This doesn't look like valid yaml. In fact
```
$ curl -s https://raw.githubusercontent.com/ingydotnet/zilla-dist-pm/c8a6097070d5136ea6c14551790dafc91b4aba28/.travis.yml | ysh
YAML::XS::Load Error: The problem:

    did not find expected comment or line break

was found at document: 1, line: 53, column: 18
while scanning a block scalar at line: 53, column: 17
```

File: .travis.yml (hunk 1 above)
Comment (mohawk2) [reply .travis.yml]: Done

File: .travis.yml (hunk 2 above)
Comment (mohawk2) [reply .travis.yml]: Thanks! Fixed. No idea why Travis worked with it.

**Pull request merged**